### ISSUE TITLE
Equation numbering

### ISSUE THREAD

**Issue by JRussellHuffman** [Qiskit/platypus#983]
In jupyter notebooks and LaTex, it is standard to have equations numbered to the right of an equation in parenthesis (see screenshot).
<img width="675" alt="Screen Shot 2022-05-03 at 9 31 52 AM" src="https://user-images.githubusercontent.com/7633881/166473575-584e4863-6150-4540-b341-3f891f14a733.png">

This is currently missing from Platypus and we should add it.

**Comment by jhwatrous** [comment]
Related issues: #980 #545 #22

**Comment by techtolentino** [comment]
Documentation on equation numbering in Mathjax

https://docs.mathjax.org/en/latest/input/tex/eqnumbers.html

**Comment by techtolentino** [comment]
For what it's worth, we do have equation tagging working on this page:

https://learn.qiskit.org/course/ch-applications/flexible-representation-of-quantum-images-frqi

However, authors are manually adding a `{\tag(1.1)}` to these equations. 

The same pattern can be see in this notebook:

https://github.com/Qiskit/platypus/blob/main/notebooks/ch-applications/quantum-edge-detection.ipynb

**Comment by techtolentino** [comment]
@JRussellHuffman @jhwatrous 

Question for either of you:

Seeing that on the [Flexible Representation of Quantum Images (FRQI)](https://learn.qiskit.org/course/ch-applications/flexible-representation-of-quantum-images-frqi#mjx-eqn-eq%3AFRQI_state) page, the `{\tag ()}` is working, and that author's also have the ability to link to a tab (shown in video below), would you still consider this a bug, or a content issue?



https://user-images.githubusercontent.com/6276074/173905865-d748fc0a-7e5d-459d-bf3c-a38120c9987f.mov

**Comment by JRussellHuffman** [comment]
I'm not sure I'm the right person to answer whether or not this is a bug or content issue. It sounds like a bug in that the functionality exists, but isn't working properly because it isn't rendering (by default). But to John's point, we are unblocked, which is the important part.

**Comment by techtolentino** [comment]
@JRussellHuffman - just pointing out that this functionality is [turned off by default](https://docs.mathjax.org/en/latest/input/tex/eqnumbers.html#:~:text=This%20functionality%20is%20turned%20off%20by%20default%2C).

I _did_ try configuring our app to turn it on by default, but it wasn't as easy as the docs say it is. In other words, it'll take a bit more time to figure out why the patch that I wrote, wasn't working.

FWIW, we're not using MathJax straight out of the box. We're using MathJax by way of Mathigon, which explains why I think we'll need a bit more investigating. 

Since the tagging does work for John and other authors, we can close this issue. To continue the progress for this feature, I opened a new one around [enabling Automatic Equation Numbering for All equations](https://github.com/Qiskit/platypus/issues/1189).

Thank you all 👏
